iOS Client/DevotionsController/MBDevotionsViewController.swift

```diff
@@ -12,11 +12,17 @@ import ReSwift
 class MBDevotionsViewController: UIViewController, StoreSubscriber, UITableViewDelegate, UITableViewDataSource {
 
     @IBOutlet weak var tableView: UITableView!
-    var devotions: [MBDevotion] = []
+    var store: MBStore!
+    var devotions: [LoadedDevotion] = []
+    var cellReusableId: String = "DevotionTableViewCell"
```

Comment:
i've usually seen `reuseIdentifier`, but I'm not gonna split hairs over it (and the value would be something more like `DevotionCellReuseIdentifier`)